cmd/podman/machine/init.go

```diff
@@ -106,7 +106,13 @@ func init() {
 	_ = initCmd.RegisterFlagCompletionFunc(IgnitionPathFlagName, completion.AutocompleteDefault)
 
 	rootfulFlagName := "rootful"
+
 	flags.BoolVar(&initOpts.Rootful, rootfulFlagName, false, "Whether this machine should prefer rootful container execution")
+
+	flags.BoolVar(
+		&initOpts.QemuStatic,
+		"with-foreign-arch", false,
```

Comment:
I am not sure whether the naming was debated already. But other commands just use `--arch` (e.g., `podman run`), so I'd prefer to keep the naming consistent and rename it to `--arch`.

Reply:
I agree it's out of step with the style in which the rest of the options are named.
I am listing below some possibilities I can think of.  How do you [plural] feel about them?

- `--cross`
- `--cross-arch`
- `--non-native`
- `--emulation`
- `--emulate`

FWIW, the two that start with `--cross` are my favourite (evocative of cross-compilation).

Incidentally, do you [plural] know off the top of your head if `spf13/cobra` does "unique prefix" option matching (i.e., if there's an option spelled `--foobar`, and no other option starts with `--foo`, will it accept just `--foo` in place of `--foobar`?)